decidim-core/app/cells/decidim/upload_modal/files.erb

```diff
@@ -1,11 +1,23 @@
-<div class="dynamic-uploads <%= with_title %> upload-container-for-<%= attribute %>">
-  <%= tag.strong label unless has_title? %>
-  <div class="<%= actions_wrapper_class %>">
-    <div class="active-uploads">
+<div class="upload-modal__files-container upload-container-for-<%= attribute %> <%= with_title %>">
+  <div>
+    <%= tag.label label %>
+
+    <% if attachments.blank? && !has_title? && uploader_default_image_path(attribute).present? %>
+      <%= image_tag uploader_default_image_path(attribute) %>
```

Comment:
Should we have some way to remove existing files? Or is this something specific only to the verifications?

What if we use this same modal e.g. for the avatar image and I have already uploaded an avatar before and want to remove it later on without adding a new avatar. Is there some way to achieve this right now?

Reply:
You are right, currently there is no way to remove the existing image if already saved, the existing files aren't shown in the modal to remove them, we're going to fix it.

Reply:
I've included that feature about removing the current image (sweat & tears). Deployed in staging